cl_manycore/libraries/bsg_manycore_tile.cpp

```diff
@@ -55,51 +60,259 @@ int hb_mc_tile_unfreeze (uint8_t fd, uint8_t x, uint8_t y) {
                                     x, y, HB_MC_PACKET_OP_REMOTE_STORE);
 	if (hb_mc_fifo_transmit(fd, HB_MC_FIFO_TX_REQ, &unfreeze) != HB_MC_SUCCESS)
 		return HB_MC_FAIL;
-	else
-		return HB_MC_SUCCESS;
+
+	#ifdef DEBUG
```

Comment:
please use the print macros defined in `bsg_manycore_printing.h`

Reply:
This is great! I will change all these statements, thanks for pointing it out!